packages/flutter_tools/test/integration.shard/hot_reload_test.dart

```diff
@@ -131,6 +131,7 @@ void main() {
     final Completer<void> sawDebuggerPausedMessage2 = Completer<void>();
     final StreamSubscription<String> subscription = _flutter.stdout.listen(
       (String line) {
+        print('[LOG]:"$line"');
```

Comment:
Would the stderr be useful as well?

Reply:
I haven't seen any unexpected stderrr logs, this is more to figure out where it is getting stuck